backend/agent_adapter.py

```
@@ -70,220 +65,101 @@ def __init__(self, vector_store_path: Optional[Path] = None, model: Optional[str
-        if self.degraded_components:
-            component_names = [d["component"] for d in self.degraded_components]
+        # If no multi_agent config, load from extension file
+        if not multi_agent_config:
+            extension_path = project_root / "config_multi_agent_extension.json"
+            if extension_path.exists():
+                try:
+                    with open(extension_path) as f:
+                        extension_config = json.load(f)
+                        multi_agent_config = extension_config.get("multi_agent", {})
+                except Exception as e:
+                    logger.error(f"Could not load config_multi_agent_extension.json: {e}")
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)